x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -132,8 +135,8 @@ export const TransactionDistribution: FunctionComponent<Props> = (
   );
 
   const xMax = d3.max(buckets, d => d.x) || 0;
-  const timeFormatter = getTimeFormatter(xMax);
-  const unit = timeUnit(xMax);
+  const timeFormatter = getDurationFormatter(xMax);
+  const unit = durationUnit(xMax);
```

Comment:
Can you add the `get` prefix:

```suggestion
  const unit = getDurationUnit(xMax);
```

This way we can avoid potential naming conflicts like:
```js
const durationUnit = durationUnit(xMax)
```